src/view/ChewingEditor.h

```diff
@@ -24,6 +24,8 @@
 #include <QFileDialog>
 #include <QMainWindow>
 
+#include <config.h>
```

Comment:
indeed is better, less header dependency means less work for recompilation (when code changes) and less header inclusion.